src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml.cs

```diff
@@ -45,6 +45,7 @@ static void UpdateVisibility(DependencyObject d, DependencyPropertyChangedEventA
         void Button_Click(object sender, RoutedEventArgs e)
```

Comment:
We should probably change this to something a little more descriptive? I'm :thought_balloon: `Dismiss_Click`?